### PR TITLE
Remove contrib crates.

### PR DESCRIPTION
**Pull request by hdost**
They have now been added in opentelemetry-rust-contrib

Relates #841

Fixes #
Design discussion issue (if applicable) #

## Changes

Please provide a brief description of the changes here.

## Merge requirement checklist

* [ ] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-rust/blob/main/CONTRIBUTING.md) guidelines followed
* [ ] Unit tests added/updated (if applicable)
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial, user-facing changes
* [ ] Changes in public API reviewed (if applicable)

### CODE REVIEW

**Comment by hdost** [comment]
Added into other repo here: https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/2

**Comment by hdost** [comment]
> Added into other repo here: [open-telemetry/opentelemetry-rust-contrib#2](https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/2)

This has been merged.

**Comment by hdost** [comment]
@open-telemetry/rust-approvers What do you think about reviewing this, this should hopefully ensure less patches that need to be cherry-picked into the contrib repo.

**Review by cijothomas — APPROVED** [review]

**Review by lalitb — APPROVED** [review]

File: examples/traceresponse/Cargo.toml [hunk 1]
```diff
@@ -1,25 +0,0 @@
-[package]
-name = "traceresponse"
```
Comment (lalitb) [inline examples/traceresponse/Cargo.toml:2]: also remove `traceresponse` example reference from [ README.md](https://github.com/open-telemetry/opentelemetry-rust/blob/v0.21.0/examples/README.md)

**Pull request merged**